mmv1/templates/terraform/examples/dns_managed_zone_basic.tf.erb

```diff
@@ -1,3 +1,4 @@
+#[START dns_managed_zone_basic]
```

Comment:
```suggestion
# [START dns_managed_zone_basic]
```
it looks like we do not detect the sample for cloud docs unless it it has a space after the `#`. I can account for that in our code, but for this PR let's add the space.